src/constants/networks.js

```diff
@@ -190,6 +190,18 @@ export const defaultNetworks = {
   },
 }
 
+export const SWAP_RBTC_PROXY_ADDRESS = {
```

Comment:
you don't need this whole new object, you just need the address attached to the rsk testnet network config.